src/index.js

```diff
@@ -0,0 +1,150 @@
+let allCityDetails
+
+/**
+ * This function will change the icons for particular hour based on temperature value
+ * @param {string} temp current temperature
+ * @param {number} i increment value for accessing each classes iteretively
+ */
+function changeIconImage (temp, i) {
+  temp = parseInt(temp.slice(0, 2))
+  console.log(temp)
+  const iconImage = document.getElementsByClassName('top-ic')[i]
+  if (temp >= 23 && temp <= 29) { iconImage.src = '../Assets/HTML&CSS/Weather_Icons/cloudyIcon.svg' } else if (temp < 18) { iconImage.src = '../Assets/HTML&CSS/Weather_Icons/rainyIcon.svg' } else if (temp <= 22 && temp >= 18)iconImage.src = '../Assets/HTML&CSS/Weather_Icons/windyIcon.svg'
+  else iconImage.src = '../Assets/HTML&CSS/Weather_Icons/sunnyIcon.svg'
+}
+
+/**
+ *This function will change the top hourly temperature value for next 5 hours
+ * @param {object} currentCity This object holds the current city value
+ * @param {string} hour current hour of the city
+ * @param {string} state current Meridiem state
+ */
+function changeTopScroll (currentCity, hour, state) {
+  const topContainer = document.getElementsByClassName('top-end')[0]
+  topContainer.innerHTML = ''
+  for (let i = 0; i < 5; i += 1) {
+    if (i === 0) {
+      const element = document.createElement('div')
+      element.classList.add('topscroll-items')
+      element.innerHTML = `
+      <div class="item-time">NOW</div>
+      <div class="pipe">|</div>
```

Comment:
These types of template tags can be stored in a different place. Let's say a separate function saying getCityTimeHTMLTemplate(somedata attributes) will return this. This is just to increase the readability.